db/settings.go

```diff
@@ -11,11 +20,16 @@ type Settings struct {
 	Name               string `yaml:"NAME"`
 	MaxOpenConnections int    `yaml:"MAX_OPEN_CONNECTIONS"`
 	MaxIdleConnections int    `yaml:"MAX_IDLE_CONNECTIONS"`
+	SSLMode            string `yaml:"SSL_MODE"`
 }
 
 // BuildConnectionString builds the connection string to the database - for now same as reader
-func (app *Settings) BuildConnectionString(withSearchPath bool, sslMode SSLMode) string {
-	cs := fmt.Sprintf("user=%s password=%s dbname=%s host=%s port=%s sslmode=%s",
+func (app *Settings) BuildConnectionString(withSearchPath bool) string {
+	sslMode := SSLModeDisable
+	if app.SSLMode != "" {
+		sslMode = app.SSLMode
+	}
+	cs := fmt.Sprintf("user=%s password=%s dbname=%s host=%s port=%s =%s",
```

Comment:
missing `sslmode=` ?
```suggestion
	cs := fmt.Sprintf("user=%s password=%s dbname=%s host=%s port=%s sslmode=%s",
```